docs/serving/revision-gc.md

```diff
@@ -0,0 +1,19 @@
+# Revision Garbage Collection
```

Comment:
```suggestion
# Revision garbage collection
```

Reply:
I'm not sure there's a need to duplicate this information here - IMO it would be enough to have the admin guide configmap based docs, and then have a section in the developer guide with the per-revision docs, rather than just duplicating info in the Knative Serving section that we've been trying to move into dev vs admin docs.

I won't block the PR on this though since the structure is already a mess, just something to think about.

Reply:
Yeah - I'm mixed on it - but made the separate section since the per-revision option may be available to end-users